pkg/authorization/api/v1beta1/types.go

```diff
@@ -16,15 +16,15 @@ import (
 // PolicyRule holds information that describes a policy rule, but does not contain information
 // about who the rule applies to or which namespace the rule applies to.
 type PolicyRule struct {
-	// Deny is true if any request matching this rule should be denied.  If false, any request matching this rule is allowed.
-	Deny bool `json:"deny"`
 	// Verbs is a list of Verbs that apply to ALL the ResourceKinds and AttributeRestrictions contained in this rule.  VerbAll represents all kinds.
 	Verbs []string `json:"verbs"`
 	// AttributeRestrictions will vary depending on what the Authorizer/AuthorizationAttributeBuilder pair supports.
 	// If the Authorizer does not recognize how to handle the AttributeRestrictions, the Authorizer should report an error.
 	AttributeRestrictions kruntime.RawExtension `json:"attributeRestrictions"`
-	// ResourceKinds is a list of kinds this rule applies to.  ResourceAll represents all kinds.
-	ResourceKinds []string `json:"resourceKinds""`
+	// ResourceKindsDeprecated is a list of resources this rule applies to.  ResourceAll represents all resources.
+	ResourceKindsDeprecated []string `json:"resourceKinds""`
```

Comment:
> Internal types can change on the fly.  In external types you add the new field, and have conversion do the if set { //new value } else { //old value }

@liggitt I'm pretty sure that's what I did except adding Deprecated to the external name.  Sounds like I just remove the Deprecated from the name and we're good?